notebooks/003-hello-segmentation/003-hello-segmentation.ipynb

```diff
@@ -0,0 +1,191 @@
+    ")\n",
+    "exec_net = ie.load_network(net, \"CPU\")\n",
+    "\n",
+    "output_layer_ir = next(iter(exec_net.outputs))\n",
+    "input_layer_ir = next(iter(exec_net.input_info))"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Load an Image"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "# The segmentation network expects images in RGB format\n",
+    "image = cv2.cvtColor(cv2.imread(\"data/coco_cross.png\"), cv2.COLOR_BGR2RGB)\n",
+    "\n",
+    "# N,C,H,W = batch size, number of channels, height, width\n",
+    "N, C, H, W = net.input_info[input_layer_ir].tensor_desc.dims\n",
+    "\n",
+    "# OpenCV resize expects the destination size as (width, height)\n",
+    "input_image = cv2.resize(image, (W, H))  \n",
+    "# reshape to network input shape\n",
+    "input_image = np.expand_dims(\n",
+    "    input_image.transpose(2, 0, 1), 0\n",
+    ")  \n",
+    "plt.imshow(image)"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Do Inference"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "# Run the infernece\n",
+    "result = exec_net.infer(inputs={input_layer_ir: input_image})\n",
+    "result_ir = result[output_layer_ir]\n",
+    "\n",
+    "# Prepare data for visualization\n",
+    "result_ir_idx = np.argmax(result_ir, axis=1)\n",
+    "plt.imshow(result_ir_idx[0])\n"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Prepare data for visualisation"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "# Define colormap, each color represents other object type \n",
+    "colormap = np.array([[255, 0, 122], [0, 255, 0], [255, 0, 0], [255, 255, 0]])\n",
+    "\n",
+    "# Define how transparent will be mask on photo\n",
+    "alpha = 0.15\n",
+    "\n",
+    "# Use function from notebook_utils.py to create mask\n",
+    "mask = segmentation_map_to_image(result_ir_idx, colormap)\n",
+    "\n",
+    "# Transpose image back to HWC(height, width, channels) layout\n",
+    "input_image_t = np.squeeze(input_image).transpose(1, 2, 0)\n",
+    "\n",
+    "# Create image with mask put on\n",
+    "image_with_mask = cv2.addWeighted(mask, alpha, input_image_t, 1 - alpha, 0)\n"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Visualise data"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "# Define titles with images\n",
+    "data = {'Segmentation': mask, \"Base Photo\": input_image_t, \"Masked Photo\": image_with_mask}\n",
+    "\n",
+    "# Create subplot to visualise images\n",
```

Comment:
FYI: we use American English spelling, so "visualize" etc. We will add this to the contribution guide.